rnd/infra/terraform/environments/dev.tfvars

```diff
@@ -10,3 +10,37 @@ node_pool_name  = "dev-main-pool"
 machine_type    = "e2-medium"
 disk_size_gb    = 100
 static_ip_names = ["agpt-server-ip", "agpt-builder-ip", "auth-ip"]
+
+
+service_accounts = {
+  "dev-agpt-server-sa" = {
+    display_name = "AutoGPT Dev Server Account"
+    description  = "Service account for agpt dev server"
+  }
+}
+
+workload_identity_bindings = {
```

Comment:
Workload identity so a kubernetes account can act with the same perms as a GCP service account, _without_ having to pass around credentials as json files